### PR TITLE
fix: flash orange mpm module

### LINKED ISSUE #4048: Cannot flash external Orange base multi module
### Is there an existing issue for this problem?

- [X] I have searched the existing issues

### What part of EdgeTX is the focus of this bug?

Transmitter firmware

### Current Behavior

Trying to flash mm-orangerx-aetr-green-inv-v1.3.3.20.bin result in NEEDS FILE opentx-inv

### Expected Behavior

Flash should be possible

### Steps To Reproduce

Put mm-orangerx-aetr-green-inv-v1.3.3.20.bin on sd, try to flash external module

### Version

2.9.0

### Transmitter

Radiomaster Boxer

### Operating System (OS)

_No response_

### OS Version

_No response_

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by 3djc**
Fix issue where orange mpm modules firmware was not accepted

This fixes #4048

### CODE REVIEW

File: radio/src/io/multi_firmware_update.h [hunk 1]
```diff
@@ -81,9 +81,19 @@ class MultiFirmwareInformation
...
-              telemetryType == FIRMWARE_MULTI_TELEM_MULTI_TELEMETRY);
+      switch(boardType) {
+        case FIRMWARE_MULTI_STM:
+        case FIRMWARE_MULTI_AVR:
+          return (telemetryInversion && optibootSupport && bootloaderCheck &&
+                  telemetryType == FIRMWARE_MULTI_TELEM_MULTI_TELEMETRY );
+
+        case FIRMWARE_MULTI_ORX:
+          return (telemetryInversion && bootloaderCheck &&
+                  telemetryType == FIRMWARE_MULTI_TELEM_MULTI_TELEMETRY );
```
Comment (raphaelcoeffic) [inline radio/src/io/multi_firmware_update.h:92]: So for all modules something like that? 
```c
return ( telemetryInversion && bootloaderCheck &&
              (boardType == FIRMWARE_MULTI_ORX || optibootSupport) &&
              telemetryType == FIRMWARE_MULTI_TELEM_MULTI_TELEMETRY );
```

Comment (3djc) [reply radio/src/io/multi_firmware_update.h:92]: You could write it like that, I just don't find it easy to read. Can change if you find it clearer this way

Comment (raphaelcoeffic) [reply radio/src/io/multi_firmware_update.h:92]: Just shorter 😬

**Comment by pfeerick** [comment]
So what's the latest for this PR? Looks fine to me, and I'm not fussed about the style of the `isMultiExternalFirmware()` test... it seems more readable in long form but works either way.

**Comment by 3djc** [comment]
Waiting for answers from Pascal

**Pull request closed without merging**